website/docs/reference/feature-toggles.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Feature Toggles
+---
+
+Feature toggles are the central concept that we built Unleash around. In Unleash, feature toggles are organized within [projects](../user_guide/projects.md). Feature toggles can have different [activation strategies](../user_guide/activation-strategies.md) for each of their project's [environments](../user_guide/environments.md), and can also be given [variants](../advanced/feature-toggle-variants.md) to facilitate A/B testing.
+
+## Configuration options
+
+Each feature toggle has the following configuration options
+
+| Option | Required? | Default value | Description |
+| --- | --- | --- | --- |
+| name | Yes | N/A | The feature toggle's name. Must be URL-friendly according to [section 2.3 of RFC 3986](https://www.rfc-editor.org/rfc/rfc3986#section-2.3) and must be unique within your Unleash instance. |
+| feature toggle type | Yes | Release | The [feature toggle's type](../advanced/feature-toggle-types.md). |
+| project | Yes | Your current project | The [project](../user_guide/projects.md) that should contain the feature toggle. |
+| description | No | N/A | A short text to describe the feature toggle's purpose. |
+| enable impression data | Yes | No | Whether to enable [impression data](../advanced/impression-data.md) for this toggle or not. |
+
+## Environments
+
+You will probably not want to use the same configuration to enable a toggle in development as you do in production. That's why feature toggles have different activation strategy configurations for each environment.
+
+You can enable and disable a toggles environments independently. You can only choose environments that are active for the current project. When you disable a toggle's environment, it will always evaluate to false in that environment. When you enable an environment, the toggle will evaluate to true or false depending on its activation strategies.
+
```

Comment:
Yeah, so this is one of the places where I need some real help. I ... find it hard to articulate. Do you enable a toggle in an environment or do you enable an environment for a toggle? Do toggles contain environments or the other way around?

Reply:
So that's a really good question and hard to answer as well! I'm not sure there's a true answer in any real sense because from the code's perspective, toggles aren't owned by environments and environments aren't owned by toggles, they're just related. Which is pretty distinct from something like projects where there definitely is an ownership relationship. That's probably the mental model I find most useful at the moment - that it's the toggle-environment relationship that can be enabled or disabled. I would express this as something like 

"You can enable and disable a toggle in different environments independently", which just avoids ownership conversation completely (needs some rephrasing but the concept feels correct)

Reply:
Love it! I've addressed your suggestions in cd6a70080612ecc63dcefa515bf0af9b897dbf97. What do you reckon?